packages/client/src/features.ts

```diff
@@ -1,3 +1,16 @@
+import { SetFeatureFlag } from '@openshift-console/dynamic-plugin-sdk';
+import * as _ from 'lodash-es';
+
 export { default } from '@odf/core/redux';
 export * from '@odf/core/redux';
 export { detectRGW } from '@odf/core/features';
+
+const INTERNAL_MODE_PRESENT = 'ODF_INTERNAL_MODE_PRESENT';
+
+export const detectInternalMode = async (setFlag: SetFeatureFlag) => {
```

Comment:
does return type need to be `Promise<void>` ? what's the need of async ??
```suggestion
export const detectInternalMode = (setFlag: SetFeatureFlag) => {
```